pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java

```diff
@@ -1212,12 +1212,15 @@ void sendComplete(final Exception e) {
                     TimeoutException te = (TimeoutException) e;
                     long sequenceId = te.getSequenceId();
                     long ns = System.nanoTime();
+
+                    String firstSentString = firstSentAt == 0 ? "never" : String.format("%d ns ago", ns - firstSentAt);
+                    String lastSentString = lastSentAt == 0 ? "never" : String.format("%d ns ago", ns - lastSentAt);
```

Comment:
Nit/qs: Some of the class variables in OpSendMsg() (batchSizeByte, totalChunks) are init-ed to 0, while others (firstsentAt, lastSentAt) are not. What makes sense from the POV of uniformity?

Reply:
In java numeric primitives are by default initialized to 0 so explicitly initializing numeric primitive types to zero is superfluous.  I did not want to change more existing code that was not related to this PR.